docs/src/components/landing/Hero.js

```diff
@@ -49,7 +49,7 @@ export default function Hero() {
         </Typography>
         <GetStartedButtons
           sx={{ justifyContent: 'center' }}
-          installation={'npx create-toolpad-app'}
+          installation="npx create-toolpad-app@latest"
```

Comment:
If we can trust https://stackoverflow.com/questions/73184553/npx-does-not-look-for-latest-version-of-package